src/odemis/acq/fastem.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on 19 Apr 2021
+
+Copyright © 2021 Philip Winkler, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+from odemis import model
+
+
+class FastEMROA(object):
+    """ Representation of a FastEM acquisition region. """
+
+    def __init__(self, name, coordinates, calibration):
+        self.name = model.StringVA(name)
+        self.coordinates = model.TupleContinuous(coordinates,
+                                                 range=((-1, -1, -1, -1), (1, 1, 1, 1)),
+                                                 cls=(int, float))
+        self.calibration = model.VigilantAttribute(calibration)  # ROC
+
+
+class FastEMROC(object):
+    """ Representation of a FastEM parameters region. """
+
+    def __init__(self, number, coordinates):
+        self.number = model.IntVA(number, readonly=True)  # cannot be changed after initialization
+        self.coordinates = model.TupleContinuous(coordinates,
+                                                 range=((-1, -1, -1, -1), (1, 1, 1, 1)),
+                                                 cls=(int, float))
+        self.parameters = None  # Calibration object with all relevant parameters
+
+
+# TODO: replace fake testing functions with actual acquisition
+import time
+executor = model.CancellableThreadPoolExecutor(max_workers=1)
```

Comment:
Maybe mark it as private as `_executor`. I'd also mention in a comment that it's single object, independently of how many times the module is loaded.